geowrangler/vector_zonal_stats.py

```diff
@@ -339,9 +339,9 @@ def validate_data_quadkey(data, data_quadkey_column, min_zoom_level):
 
 def quadkey_to_tile(quadkey, zoom_level):
```

Comment:
Let's add an exception if the zoom level is greater than the current quad key. Can we also rename the function to something else? something like `set_quadkey_zoomlevel` or `get_quadkey_parent`

Reply:
ok.

Reply:
i dont think it raises a exception! just wanted to add an exception in case other people use this function and expect it work

```
In [2]: zoom_level=22

In [3]: quadkey = '1'

In [4]: quadkey[:zoom_level]
Out[4]: '1'
```

Reply:
Hi @jtmiclat my solution is to just remove the function since I realized that all its doing is doing a substring within the context of the create_bingtile_zonal_stats and since there is already a validation method that checks that the data zoom level should be > or equal to the aoi zoom level I don't even need to check for exceptions.